pygeoapi/__init__.py

```diff
@@ -30,15 +30,30 @@
 __version__ = '0.6.0'
 
 import click
-from pygeoapi.flask_app import serve
 from pygeoapi.openapi import generate_openapi_document
 
 
-@click.group()
-@click.version_option(version=__version__)
-def cli():
-    pass
+cli = click.Group()
+cli.version = __version__
+
+
+@cli.command()
+@click.option('--flask', 'server', flag_value="flask", default=True)
```

Comment:
does `default=True` mean that the `server` param is `flask` by default ?

Reply:
Yes it does, see answer above